src/panel/direction/RoutesList/Route/RoadMap/WalkLeg/index.jsx

```diff
@@ -7,6 +7,26 @@ import LegLine from '../LegLine';
 import classnames from 'classnames';
 import { Chevron } from 'src/components/ui';
 
+const stringifyManeuver = maneuver => {
+  const stringifyModifier = {
+    'sharp left': _('Turn left', 'direction'),
```

Comment:
We will grab the object keys from Directions appTypes enum.
So the prerequisite is to port this one in TSX.